### PR TITLE
Lab 1

### PR DESCRIPTION
**Pull request by DOCtorActoAntohich**
Lab 1
lab1

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# bad.
+.idea
```
Comment (ntdesmond) [inline .gitignore:2]: $\text{How bad: Bad}$

File: PYTHON.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+## Frameworks
+
+I chose FastAPI because:
+- It is _fast_, simple, and easy to use.
+- It's asynchronous.
+- Its small size fits this task best.
+- I have some experience with it.
+- Type validation for bigger projects (it's a must, otherwise it's hell).
+
+It cas a couple of problems that are negligible for the task:
```
Comment (ntdesmond) [inline PYTHON.md:18]: typo

**Review by ntdesmond — APPROVED** [review]
Looks well but fix the typo please

File: PYTHON.md (hunk 2 above)
Comment (DOCtorActoAntohich) [reply PYTHON.md:18]: fnuuy

File: app_python/main.py [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+
+@app.exception_handler(InvalidTimeZoneError)
+async def on_invalid_id_error(_request: StarletteRequest, exc: InvalidTimeZoneError):
+    return JSONResponse({"error_details": f"{exc}"}, status_code=400)
+
+
+@app.get("/")
+async def read_root() -> str:
+    return await get_time_in("Europe/Moscow")
```
Comment (smthngslv) [inline app_python/main.py:19]: Better to create a function. Do not use endpoints inside endpoints

File: PYTHON.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+- Can't build a beautiful UI.
```
Comment (smthngslv) [inline PYTHON.md:19]: Actually u can serve static with ur UI

File: app_python/main.py [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+from app_python.errors import InvalidTimeZoneError
+from app_python.use_case import CurrentTimeInTimeZoneUseCase
+
+
+app = FastAPI()
+
+
+@app.exception_handler(InvalidTimeZoneError)
+async def on_invalid_id_error(_request: StarletteRequest, exc: InvalidTimeZoneError):
+    return JSONResponse({"error_details": f"{exc}"}, status_code=400)
```
Comment (smthngslv) [inline app_python/main.py:14]: u can use starlette.status Enum instead of numbers

**Review by smthngslv — APPROVED** [review]
Overall, the lab done good

**Pull request closed without merging**